### ISSUE TITLE
eliminate false positives 

### ISSUE THREAD

**Issue by judell** [hypothesis/h#2240]
This was fallout from https://github.com/hypothesis/h/pull/2184. Discussion [here](https://groups.google.com/a/list.hypothes.is/d/topic/dev/E7_AnEU6Reg/discussion)

**Comment by judell** [comment]
Retitled to remove assumption about query string disambiguation.

An example of the problem: https://hypothes.is/blog/synchronizing-annotations-between-local-and-remote-pdfs/ displays "false positives" including https://hypothes.is/a/0CQYjPAUSHuFsqVxE67asg which targets http://hypothes.is/?p=1761&preview=true

An observation: https://hypothes.is/blog/synchronizing-annotations-between-local-and-remote-pdfs/ appears in [this list of ambiguous URLs](https://gist.github.com/judell/b287a63867e4123aded0):
```
Annotations on these URLs share a normalized URL but have no other metadata links:
hypothes.is/blog/synchronizing-annotations-between-local-and-remote-pdfs
	https://hypothes.is/blog/synchronizing-annotations-between-local-and-remote-pdfs/
		id: z6im1evITGWtoHr1NNuxOw updated: 2015-04-03T11:24:23.241522+00:00
	https://hypothes.is/blog/synchronizing-annotations-between-local-and-remote-pdfs/#
		id: 1ZJqA0L1TnyfDgmZTxRQ3g updated: 2015-04-03T20:19:53.070769+00:00
```

The script that produced that list is [here](https://gist.github.com/tilgovi/3db47748cdee9bd1cbb3).

**Comment by judell** [comment]
Tagged: false_positives

**Comment by nickstenning** [comment]
Currently working on this as part of sorting out our URI normalisation.

**Comment by judell** [comment]
:heart:

**Comment by tilgovi** [comment]
I re-opened, but then closed this, because I think #2440 is now fixed and we can try to roll out this feature.

**Comment by tilgovi** [comment]
Re-opening again to track the remaining issues.

**Comment by tilgovi** [comment]
https://github.com/hypothesis/h/issues/2484

**Comment by nickstenning** [comment]
Closed by #2459.

**Comment by tilgovi** [comment]
That doesn't seem right. :-/. Closed by #2459?

**Comment by tilgovi** [comment]
#2549
